code/index.html

```diff
@@ -20,24 +20,37 @@
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1>Does the person have...?</h1>
 
       <select id="questions">
         <optgroup label="hair">
+          <option>START HERE!!!</option>
+          <option value="black">black hair</option>
+          <option value="blonde">blonde hair</option>
           <option value="brown">brown hair</option>
-          <option value="yellow">yellow hair</option>
+          <option value="ginger">ginger hair</option>
+          <option value="hidden">hidden hair</option>
+          <option value="different">different colored hair</option>
           <!-- Add the rest of hair colors as options  -->
         </optgroup>
         <optgroup label="eyes">
           <option value="hidden">hidden eyes</option>
+          <option value="blue">blue eyes</option>
+          <option value="green">green eyes</option>
+          <option value="brown">brown eyes</option>
           <!-- Add the rest of eye colors as options  -->
         </optgroup>
         <optgroup label="accessories">
           <option value="glasses">glasses</option>
+          <option value="sunglasses">sunglasses</option>
+          <option value="hat">hat</option>
+          <option value="tie">tie</option>
           <!-- Add the other accessory option here. (hat)  -->
```

Comment:
I like that you added more options to the different categories, it makes the game even more fun :) If I might come with a small tip it is to remove comments in the code that is no longer necessary as in this section with comments to add other accessories etc.